[filament_view] clang 14

Yocto `dunfell` with meta-clang-14

ivi-homescreen fails build with filament_view errors

```
| FAILED: plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:124:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator==(nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:125:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator!=(nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
|          ^
| 4 warnings and 11 errors generated.
| [68/207] /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc
| FAILED: plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:124:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator==(nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:125:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator!=(nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:174:10: error: no type named 'mutex' in namespace 'std'
|     std::mutex _initMutex;
|     ~~~~~^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:179:22: error: expected ';' after expression
|       std::lock_guard lock(_initMutex);
|                      ^
|                      ;
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:179:12: error: no member named 'lock_guard' in namespace 'std'
|       std::lock_guard lock(_initMutex);
|       ~~~~~^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:191:22: error: expected ';' after expression
|       std::lock_guard lock(_initMutex);
|                      ^
|                      ;
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:191:12: error: no member named 'lock_guard' in namespace 'std'
|       std::lock_guard lock(_initMutex);
|       ~~~~~^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:25:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:74:36: warning: unused parameter 'params' [-Wunused-parameter]
|       const flutter::EncodableMap& params,
|                                    ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:75:25: warning: unused parameter 'loadedTextures' [-Wunused-parameter]
|       const TextureMap& loadedTextures
|                         ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:78:32: warning: unused parameter 'materialParam' [-Wunused-parameter]
|       const MaterialParameter* materialParam,
|                                ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:79:25: warning: unused parameter 'loadedTextures' [-Wunused-parameter]
|       const TextureMap& loadedTextures
|                         ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:166:3: error: cannot initialize object parameter of type 'plugin_filament_view::shapes::BaseShape' with an expression of type 'plugin_filament_view::shapes::Plane'
|   vBuildRenderable(engine_);
|   ^~~~~~~~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:246:3: error: cannot initialize object parameter of type 'plugin_filament_view::shapes::BaseShape' with an expression of type 'plugin_filament_view::shapes::Plane'
|   vBuildRenderable(engine_);
|   ^~~~~~~~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:250:60: error: no matching member function for call to 'DebugPrint'
| void Plane::DebugPrint(const char* tag) const { BaseShape::DebugPrint(tag); }
|                                                 ~~~~~~~~~~~^~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:67:18: note: candidate function not viable: no known conversion from 'const plugin_filament_view::shapes::Plane' to 'const plugin_filament_view::shapes::BaseShape' for object argument
|     virtual void DebugPrint(const char* tag) const;
|                  ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:90:10: note: candidate function not viable: requires 0 arguments, but 1 was provided
|     void DebugPrint() const override;
|          ^
| 4 warnings and 10 errors generated.
```

Build started.  Same issues in `dunfell`
```
| FAILED: plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/sphere.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:124:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator==(nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:125:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator!=(nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/sphere.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
|          ^
| 4 warnings and 11 errors generated.
| [68/207] /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc
| FAILED: plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/x86_64-poky-linux-clang++ -DASIO_DISABLE_HANDLER_TYPE_REQUIREMENTS -DASIO_HEADER_ONLY -DASIO_NO_DEPRECATED -DASIO_NO_TYPEID -DASIO_STANDALONE -DEGL_NO_X11 -DENABLE_PLUGIN_AUDIOPLAYERS_LINUX -DENABLE_PLUGIN_COMMON -DENABLE_PLUGIN_DESKTOP_WINDOW_LINUX -DENABLE_PLUGIN_FILAMENT_VIEW -DENABLE_PLUGIN_GO_ROUTER -DENABLE_PLUGIN_NAV_RENDER_VIEW -DENABLE_PLUGIN_SECURE_STORAGE -DENABLE_PLUGIN_URL_LAUNCHER -DNDEBUG -DRAPIDJSON_HAS_STDSTRING -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/config -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/shell/platform/homescreen/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/shell -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/.. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/public -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/flutter/shell/platform/common/client_wrapper/include/flutter -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/rapidjson/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/waypp/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build/third_party/waypp/waypp/protocols -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/build -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/common/. -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/audioplayers_linux/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/glib-2.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/glib-2.0/include -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/gstreamer-1.0 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/orc-0.4 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libsecret-1 -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/libmount -I/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/blkid -isystem /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/tomlplusplus/include -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0=/usr/src/debug/ivi-homescreen/2.0-r0                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden -target x86_64-poky-linux  -m64 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mlittle-endian -Qunused-arguments -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot -stdlib=libc++ -DNDEBUG -flto=thin -fPIC -fvisibility-inlines-hidden -Wtrigraphs -Wchar-subscripts -Wcomment -Wreturn-type -Wsequence-point -Wswitch -Wuninitialized -Wunused -Wswitch-bool -Wformat -Wformat-security -Wall -Wextra -Wconversion -Wsign-conversion -Wcast-align -Wunused-parameter -Winvalid-offsetof -fsigned-char -O2 -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/asio/asio/include -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/third_party/spdlog/include -stdlib=libstdc++ -isystem\"/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot-native/usr/include/c++/v1/\" -pthread -Wno-conversion -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament -isystem/__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/include/filament/filament -std=c++17 -MD -MT plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -MF plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o.d -o plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/entity/derived/shapes/plane.cc.o -c /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:124:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator==(nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:27:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/utils/smarter_pointers.h:125:21: error: unknown type name 'nullptr_t'; did you mean 'std::nullptr_t'?
|     bool operator!=(nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
|                     ^~~~~~~~~
|                     std::nullptr_t
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/9.5.0/../../../include/c++/9.5.0/x86_64-poky-linux/bits/c++config.h:262:29: note: 'std::nullptr_t' declared here
|   typedef decltype(nullptr)     nullptr_t;
|                                 ^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:21:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/components/derived/basetransform.h:24:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:174:10: error: no type named 'mutex' in namespace 'std'
|     std::mutex _initMutex;
|     ~~~~~^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:179:22: error: expected ';' after expression
|       std::lock_guard lock(_initMutex);
|                      ^
|                      ;
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:179:12: error: no member named 'lock_guard' in namespace 'std'
|       std::lock_guard lock(_initMutex);
|       ~~~~~^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:191:22: error: expected ';' after expression
|       std::lock_guard lock(_initMutex);
|                      ^
|                      ;
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/base/entityobject.h:191:12: error: no member named 'lock_guard' in namespace 'std'
|       std::lock_guard lock(_initMutex);
|       ~~~~~^
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:17:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.h:19:
| In file included from /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:25:
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:74:36: warning: unused parameter 'params' [-Wunused-parameter]
|       const flutter::EncodableMap& params,
|                                    ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:75:25: warning: unused parameter 'loadedTextures' [-Wunused-parameter]
|       const TextureMap& loadedTextures
|                         ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:78:32: warning: unused parameter 'materialParam' [-Wunused-parameter]
|       const MaterialParameter* materialParam,
|                                ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/./core/entity/derived/renderable_entityobject.h:79:25: warning: unused parameter 'loadedTextures' [-Wunused-parameter]
|       const TextureMap& loadedTextures
|                         ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:166:3: error: cannot initialize object parameter of type 'plugin_filament_view::shapes::BaseShape' with an expression of type 'plugin_filament_view::shapes::Plane'
|   vBuildRenderable(engine_);
|   ^~~~~~~~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:246:3: error: cannot initialize object parameter of type 'plugin_filament_view::shapes::BaseShape' with an expression of type 'plugin_filament_view::shapes::Plane'
|   vBuildRenderable(engine_);
|   ^~~~~~~~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/plane.cc:250:60: error: no matching member function for call to 'DebugPrint'
| void Plane::DebugPrint(const char* tag) const { BaseShape::DebugPrint(tag); }
|                                                 ~~~~~~~~~~~^~~~~~~~~~
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:67:18: note: candidate function not viable: no known conversion from 'const plugin_filament_view::shapes::Plane' to 'const plugin_filament_view::shapes::BaseShape' for object argument
|     virtual void DebugPrint(const char* tag) const;
|                  ^
| /__w/meta-flutter/dunfell-linux-dummy/build/tmp/work/core2-64-poky-linux/ivi-homescreen/2.0-r0/git/ivi-homescreen-plugins/plugins/filament_view/core/entity/derived/shapes/baseshape.h:90:10: note: candidate function not viable: requires 0 arguments, but 1 was provided
|     void DebugPrint() const override;
|          ^
| 4 warnings and 10 errors generated.
```

Passes to
```
| FAILED: libs/camutils/CMakeFiles/camutils.dir/src/Bookmark.cpp.o
| /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/recipe-sysroot-native/usr/bin/arm-poky-linux-gnueabi/arm-poky-linux-gnueabi-clang++ --sysroot=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/recipe-sysroot -DFILAMENT_DRIVER_SUPPORTS_VULKAN -DFILAMENT_SUPPORTS_WAYLAND -I/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/camutils/include -I/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/math/include -target arm-poky-linux-gnueabi  -mthumb -mfpu=neon -mfloat-abi=hard -mcpu=cortex-a15 -mlittle-endian -Qunused-arguments -fstack-protector-strong  -O2 -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fmacro-prefix-map=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0=/usr/src/debug/filament-vk/1.4.92-r0                      -fdebug-prefix-map=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0=/usr/src/debug/filament-vk/1.4.92-r0                      -fdebug-prefix-map=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/recipe-sysroot=                      -fdebug-prefix-map=/__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -std=c++17 -fstrict-aliasing -Wno-unknown-pragmas -Wno-unused-function -Wno-deprecated-declarations -stdlib=libc++ -fPIC -fcolor-diagnostics -fvisibility=hidden -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wno-deprecated-register -MD -MT libs/camutils/CMakeFiles/camutils.dir/src/Bookmark.cpp.o -MF libs/camutils/CMakeFiles/camutils.dir/src/Bookmark.cpp.o.d -o libs/camutils/CMakeFiles/camutils.dir/src/Bookmark.cpp.o -c /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/camutils/src/Bookmark.cpp
| In file included from /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/camutils/src/Bookmark.cpp:17:
| In file included from /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/camutils/include/camutils/Bookmark.h:22:
| In file included from /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/math/include/math/vec2.h:20:
| In file included from /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/math/include/math/TVecHelpers.h:20:
| /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/math/include/math/compiler.h:20:10: fatal error: 'type_traits' file not found
|    20 | #include <type_traits>
|       |          ^~~~~~~~~~~~~
| 1 error generated.
```
Also
```
| In file included from /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/image/src/LinearImage.cpp:17:
| /__w/meta-flutter/kirkstone-linux-dummy/build/tmp/work/cortexa15t2hf-neon-poky-linux-gnueabi/filament-vk/1.4.92-r0/git/libs/image/include/image/LinearImage.h:22:10: fatal error: 'cstdint' file not found
|    22 | #include <cstdint>
|       |          ^~~~~~~~~
```

I need to investigate this further; later

--- a/plugins/filament_view/core/entity/base/entityobject.h
+++ b/plugins/filament_view/core/entity/base/entityobject.h
@@ -16,6 +16,7 @@
 #pragma once
 
 #include <encodable_value.h>
+#include <mutex>
 #include <string>
 #include <vector>
 

--- a/plugins/filament_view/core/entity/derived/shapes/plane.cc
+++ b/plugins/filament_view/core/entity/derived/shapes/plane.cc
@@ -246,7 +246,4 @@ void Plane::createSingleSidedPlane(filament::Engine* engine_) {
   vBuildRenderable(engine_);
 }
 
-////////////////////////////////////////////////////////////////////////////
-void Plane::DebugPrint(const char* tag) const { BaseShape::DebugPrint(tag); }
-
 }  // namespace plugin_filament_view::shapes

--- a/plugins/filament_view/core/entity/derived/shapes/plane.h
+++ b/plugins/filament_view/core/entity/derived/shapes/plane.h
@@ -43,8 +43,6 @@ class Plane : public BaseShape {
     Plane(const Plane&) = delete;
     Plane& operator=(const Plane&) = delete;
 
-    void DebugPrint(const char* tag) const override;
-
     bool bInitAndCreateShape(::filament::Engine* engine_, FilamentEntity entityObject) override;
 
   private:

--- a/plugins/filament_view/core/utils/smarter_pointers.h
+++ b/plugins/filament_view/core/utils/smarter_pointers.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstddef>
 #include <memory>
 #include <stdexcept>
 #include <type_traits>
@@ -121,8 +122,8 @@ class smarter_ptr {
       return pointer_access<P>::get(_ptr) == pointer_access<P>::get(o._ptr);
     }
     bool operator!=(const smarter_ptr& o) const { return !(*this == o); }
-    bool operator==(nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
-    bool operator!=(nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
+    bool operator==(std::nullptr_t) const { return pointer_access<P>::get(_ptr) == nullptr; }
+    bool operator!=(std::nullptr_t) const { return pointer_access<P>::get(_ptr) != nullptr; }
 };
 
 //==============================================================================

